pkg/controllers/dlb/controller.go

```diff
@@ -148,6 +154,14 @@ func (c *controller) UpdateStatus(rawObj client.Object, ds *apps.DaemonSet, node
 	return updated, nil
 }
 
+func (c *controller) Upgradable() bool {
+	return c.upgradable
+}
+
+func (c *controller) SetUpgradable(upgradable bool) {
+	c.upgradable = upgradable
+}
+
```

Comment:
Would it make sense to move upgradable attribute and above 2 methods to the separate struct and then inherit controller struct from it? This way we could avoid duplicating these methods for every controller.